docs/_layouts/default.html

```diff
@@ -35,7 +35,8 @@
     <div class="daptiv-container">
         {{ content }}
 
-        <footer class="footer">
+
+        <footer class="footer" style="margin: 25px 10px 2px 10px;">
```

Comment:
The css looks like it's from a package and I could only find the 'min' version (after an admittedly brief search). I wasn't inclined to update a minified css file. :smile: